runtime/config-deployer-service/ballerina/types.bal

```diff
@@ -202,6 +202,11 @@ public type Authentication record {|
     boolean enabled = true;
 |};
 
+public type HeaderModifierPolicy record {
```

Comment:
Let's add documentation comments for all public records

Reply:
Fixed in https://github.com/wso2/apk/commit/d0e33c3c0dcbc8a915ee237ff0900a26148978d5